scripts/cpp/fixInludesCase.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/sh
+
+# BSD Zero Clause License
+#
+# Copyright (c) 2025 zzambers
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+# REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+# AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+# INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+# LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+# OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+# PERFORMANCE OF THIS SOFTWARE.
+
+set -eu
+
+scriptName="$0"
+
+printHelp() {
+cat << EOF
+USAGE:
+${scriptName} [OPTION]... SRC_DIR
+
+Fixes filename case include directives of source files found in SRC_DIR,
+as necessary. It also fixes slashes ( \\ -> / ).
+
+Details:
+Script tries to find included file in include paths. (first relative to source
+file's directory, then in specified include paths). First it tries
+case-sensitive search in all paths. If that fails, it tries case-insensitive
+search in all include paths (excpet for nofix paths). If it succseeds, it fixes
+filename in include directory as necessary. If it fails no change is done.
```

Comment:
Mistake here. It should have been `If it succeeds, it fixes
filename in include directive as necessary.` I'll fix it later